client/customer-effort-score-tracks/data/actions.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import TYPES from './action-types';
+
+/**
+ * Initialize the state
+ *
+ * @param {Object} queue	initial queue
+ */
+export function setCesSurveyTracksQueue( queue ) {
+	return {
+		type: TYPES.SET_CES_SURVEY_TRACKS_QUEUE,
+		queue,
+	};
+}
+
+/**
+ * Add a new CES track to the state.
+ *
+ * @param {string} action action name for the survey
+ * @param {string} label label for the snackback
+ * @param {string} pageNow value of window.pagenow
+ * @param {string} adminPage value of window.adminpage
+ * @param {string} onsubmit_label label for the snackback onsubmit
+ */
+export function addCesSurveyTrack(
+	action,
+	label,
+	pageNow = window.pagenow,
+	adminPage = window.adminpage,
+	onsubmit_label = undefined
```

Comment:
Why explicitly set `onsubmit_label` to `undefined`?

Reply:
I think it's better for readability since it is explicit and the previous two variables also define a default value, but I don't have a strong preference for it. Do you think we should remove it?

Reply:
Of course. Since it comes after other params with defaults, it too should have a default. Nevermind!